ledger-core/v2/insolar/defaults/defaults_test.go

```diff
@@ -55,6 +56,10 @@ var cases = []tCase{
 }
 
 func TestDefaults(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

Comment:
Please use filepath package to fix the test, not to skip it

Reply:
done